src/App.tsx

```diff
@@ -0,0 +1,26 @@
+<<<<<<< HEAD
+import styled, { ThemeProvider } from "styled-components";
+import Theme from "./styles/theme";
+
+const Title = styled.h1`
```

Comment:
Please, move this to own file with the name Text.ts and save this file in src/UI. In this folder, we save all our styled-components